### ISSUE TITLE
Change repo about description

### ISSUE THREAD

**Issue by oxarbitrage** [ZcashFoundation/zcash_script#107]
Currently we have this as the description of the project:

"rust wrapper for `zcash`'s zcashconsensus library"

This looks old as there is no such zcashconsensus library. Should we change this ? @conradoplg what do you think?

**Comment by conradoplg** [comment]
Good catch, we could use the description from the README, e.g. "Rust bindings to the ECC's zcash_script C++ library."
